content/_index.md

```diff
@@ -151,6 +151,7 @@ a.linkhighlight:hover:after, a.linkhighlight:focus:after {
     overflow: auto;
     display: block;
   }
+ 
```

Comment:
Instead of adding `style` attribute for all `i` elements separately, please use CSS selector here, e.g. `h4 > i`. Ideally, replace `center` HTML element (the one with `<h2>Features</h2>` header and all the features) with `<div class="features">` and put the following CSS styles:

```css
.features {
  text-align: center;
}

.features h4 > i {
  margin-right: 10px;
}
```